framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -1022,6 +1022,37 @@ protected int getCacheSize() {
         }
     }
 
+    /**
+     * Returns an AnnotatedTypeMirror representing the annotated type of {@code elt} as a type
+     * declaration.
+     *
+     * <p>It is not possible to tell whether an element is a use of a type or a declaration, so this
+     * method differentiates the two.
+     *
+     * @param elt element whose type is requested
+     * @return the annotated type of {@code elt} as a type declaration
+     */
+    public AnnotatedTypeMirror getAnnotatedTypeAsTypeDecl(Element elt) {
+        return getAnnotatedType(elt);
+    }
+
+    /**
+     * Returns an AnnotatedTypeMirror representing the annotated type of {@code tree} as a type
+     * declaration.
+     *
+     * <p>It is possible to tell whether a tree is a type declaration, but extends and implements
```

Comment:
Why should extends/implements clauses also get the same default? This sentence is a bit confusing.